### PR TITLE
Send Email

### PR DESCRIPTION
**Pull request by nelsonic**
@SimonLab this PR sends a `welcome` email to people who authenticate with Google/GitHub. 

+ [x] Implement sending email https://github.com/dwyl/auth/issues/41

### CODE REVIEW

**Comment by nelsonic** [comment]
@SimonLab as you can see from the Codecov Coverage report, coverage is still below 100% 🙄 
but at least the code added in this PR does not push it down. 😉 

To get this working on Heroku you will need to have the two environment variables defined (_as per the instructions in the `README.md`_) you can get the `SECRET_KEY_BASE` from https://dashboard.heroku.com/apps/dwylmail/settings (_which I shared access with you earlier today_).

Let me know if anything is unclear or you have any questions.
Thanks! ☀️

File: coveralls.json [hunk 1]
```diff
@@ -3,6 +3,9 @@
       "minimum_coverage": 100
     },
     "skip_files": [
-      "test/"
+      "test/",
+      "lib/auth_mvp_web.ex",
```
Comment (nelsonic) [inline coveralls.json:7]: we aren't directly using these files so they aren't covered by tests hence adding them here.

File: config/config.exs [hunk 2]
```diff
@@ -28,3 +28,6 @@ config :phoenix, :json_library, Jason
 # Import environment specific config. This must remain at the bottom
 # of this file so it overrides the configuration defined above.
 import_config "#{Mix.env()}.exs"
+
+# https://hexdocs.pm/joken/introduction.html#usage
+config :joken, default_signer: System.get_env("SECRET_KEY_BASE")
```
Comment (nelsonic) [inline config/config.exs:33]: this needs to match the `SECRET_KEY_BASE` on dashboard.heroku.com/apps/dwylmail/settings for email to work.

File: test/auth_mvp_web/controllers/github_auth_controller_test.exs [hunk 3]
```diff
@@ -4,5 +4,9 @@ defmodule AuthMvpWeb.GithubAuthControllerTest do
   test "GET /auth/github/callback", %{conn: conn} do
     conn = get(conn, Routes.github_auth_path(conn, :index, %{code: "123", state: "http://localhost/"}))
     assert html_response(conn, 302)
+
+    # same again to exercise to the branch where person already exists:
```
Comment (nelsonic) [inline test/auth_mvp_web/controllers/github_auth_controller_test.exs:8]: you were already so close to 100% coverage that I just added these extra lines to get to 💯 
(_hope you don't mind ..._)

**Comment by nelsonic** [comment]
@SimonLab please excuse the flood of commits tidying up ...  🙄 
Hope everything is clear. 👍

**Review by SimonLab — APPROVED** [review]
Looks great, thanks @nelsonic

**Pull request merged**